pkg/deepcopy/gen.go

```diff
@@ -175,7 +175,8 @@ type ObjectGenCtx struct {
 // writeHeader writes out the build tag, package declaration, and imports
 func writeHeader(pkg *loader.Package, out io.Writer, packageName string, imports *importsList, headerText string) {
 	// NB(directxman12): blank line after build tags to distinguish them from comments
-	_, err := fmt.Fprintf(out, `// +build !ignore_autogenerated
+	_, err := fmt.Fprintf(out, `//go:build !ignore_autogenerated
+// +build !ignore_autogenerated
```

Comment:
Why do we need the old and the new syntax?

Reply:
The new syntax is not respected by previous Go versions (https://go.dev/doc/go1.16#go/build/constraint)

```
Note that //go:build lines are not supported in Go 1.16 and should not be introduced into Go programs yet.
```